pkgs/development/python-modules/btchip/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi, hidapi, pyscard, ecdsa }:
+
+buildPythonPackage rec {
+  pname = "btchip-python";
+  version = "0.1.21";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16g9l3rpxpvvkdx08mgy0ligvsfcpzdrh4hplj104cpprrbsqd6v";
+  };
+
+  buildInputs = [ hidapi pyscard ecdsa ];
```

Comment:
these are not needed during runtime?

Reply:
@FRidh there are definitely references to these three libraries in the `btchip` code.

Reply:
in order for Python modules to find each other, they need to be added as `propagatedBuildInputs`.